tests/test_video.py

```diff
@@ -0,0 +1,113 @@
+from unittest import mock
+import os.path as osp
+
+import cv2
+import numpy as np
+import pytest
+
+from datumaro.components.media import Video
+from datumaro.util.scope import on_exit_do, scoped
+from datumaro.util.test_utils import TestDir
+
+from .requirements import Requirements, mark_requirement
+
+
+def make_sample_video(path, frames=4, frame_size=(10, 20), fps=25.0):
+    """
+    frame_size is (H, W), only even sides
+    """
+
+    writer = cv2.VideoWriter(path, frameSize=tuple(frame_size[::-1]),
+        fps=float(fps), fourcc=cv2.VideoWriter_fourcc(*'MJPG'))
+
+    for _ in range(frames):
+        # Apparently, only uint8 values are supported, but not floats
+        writer.write(np.ones((*frame_size, 3), dtype=np.uint8) * 255)
+
+    writer.release()
```

Comment:
You should probably use `try`/`finally` to call `release`.

Reply:
Fixed